File: commands/members.py
import discord
from utils import db


def setup(bot: discord.Bot):
    @bot.slash_command(name="members", description="See all verified members.")
    async def members_cmd(ctx: discord.ApplicationContext):
        await ctx.defer(ephemeral=True)

        verified = []
        for member in ctx.guild.members:
            if await db.get_member_var(ctx.guild.id, member.id, "is_verified") == "true":
                verified.append(member.mention)

        text = "\n".join(verified) if verified else "None"
        await ctx.followup.send(text, allowed_mentions=discord.AllowedMentions.none())

File: commands/members.js
import { SlashCommandBuilder, MessageFlags } from "discord.js";
import { getMemberVar } from "../utils/db.js";

export default {
  data: new SlashCommandBuilder()
    .setName("members")
    .setDescription("See all verified members."),

  async execute(interaction) {
    await interaction.deferReply({ flags: MessageFlags.Ephemeral });

    const members = await interaction.guild.members.fetch();

    const verified = [];
    for (const member of members.values()) {
      if ((await getMemberVar(interaction.guild.id, member.id, "is_verified")) === "true") {
        verified.push(member.toString());
      }
    }

    const text = verified.length ? verified.join("\n") : "None";
    await interaction.followUp({ content: text, allowedMentions: { parse: [] } });
  },
};
